overrides/scripts/Nuclearcraft.zs

```diff
@@ -2,9 +2,9 @@ import crafttweaker.item.IItemStack;
 import crafttweaker.item.IIngredient;
 import crafttweaker.oredict.IOreDictEntry;
 import mods.contenttweaker.Fluid;
-
 import mods.gregtech.recipe.RecipeMap;
 
+val fluid_extractor = RecipeMap.getByName("fluid_extractor");

```

Comment:
This variable is defined but not used.

There is a globally defined reference to the fluid extractor recipe registry in [CommonVars.zs](https://github.com/Exaxxion/Omnifactory/blob/1.3-PR/overrides/scripts/CommonVars.zs) which is loaded first by CraftTweaker so we don't need to redefine the machines in each file. I had not cleaned up all of the redundant variables in the script files, so sorry if that was confusing.

Reply:
Fixed in latest commit. Better make sure at some point we don't have GT scripts that load before CommonVars though.